aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -490,6 +490,20 @@ void dot<float>(CUDABLAS_DOT_ARGTYPES(float)) {
   TORCH_CUDABLAS_CHECK(cublasSdot(handle, n, x, incx, y, incy, result));
 }
 
+template <>
+void dot<c10::complex<double>>(CUDABLAS_DOT_ARGTYPES(c10::complex<double>)) {
```

Comment:
Looks like you forget to add the template specification to `CUDABlas.h`?

Reply:
yeah that did fix the windows test failures. thanks!!